webhook/resourcesemantics/validation/controller.go

```diff
@@ -38,6 +38,7 @@ func NewAdmissionController(
 	ctx context.Context,
 	name, path string,
 	handlers map[schema.GroupVersionKind]resourcesemantics.GenericCRD,
+	callbacks map[schema.GroupVersionKind]Callback,
```

Comment:
This is ugly but can we make this the last parameter and make it variadic & thus optional? We'll need the obvious validation.

Otherwise this becomes a breaking change.

Reply:
Done!